nix/status-go/default.nix

```diff
@@ -43,8 +43,12 @@ let
         versionJSON = importJSON ../../status-go-version.json; # TODO: Simplify this path search with lib.locateDominatingFile
         versionRegex = "^v?[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+(-[[:alnum:].]+)$";
         sha256 = versionJSON.src-sha256;
+        sanitizeVersion = builtins.replaceStrings
+          [ "/" ]
+          [ "_" ];
       in rec {
-        inherit (versionJSON) owner repo version;
+        inherit (versionJSON) owner repo;
+        version = sanitizeVersion versionJSON.version;
```

Comment:
What's the difference between `version` and `versionName`?

Reply:
Actually `version` should move to the `let` area since we don't use it elsewhere. I've refactored the PR, please re-review.